state-sync/state-sync-v1/src/coordinator.rs

```diff
@@ -1656,13 +1720,15 @@ mod tests {
     use crate::{
         chunk_request::{GetChunkRequest, TargetType},
         chunk_response::{GetChunkResponse, ResponseLedgerInfo},
-        client::SyncRequest,
         coordinator::StateSyncCoordinator,
         error::Error,
         executor_proxy::ExecutorProxy,
         network::StateSyncMessage,
         shared_components::{test_utils, test_utils::create_coordinator_with_config_and_waypoint},
     };
+    use consensus_notifications::{
+        ConsensusCommitNotification, ConsensusNotificationResponse, ConsensusSyncNotification,
+    };
     use diem_config::{
```

Comment:
Not for this PR, but let's pull the tests out of this file in a different PR.  This file is > 2500 lines long!

Reply:
The beautiful of legacy code. 😆 The perspective I'm trying to keep is that because state sync v1 is going to be deprecated anyway, I'm avoiding any work that won't benefit state sync v2. Otherwise, I think it'll be really easy to burn cycles trying to fix things up